packages/admin/resources/js/app.js

```diff
@@ -70,6 +70,29 @@ window
 Chart.defaults.font.family = `'DM Sans', sans-serif`
 Chart.defaults.color = '#6b7280'
 
+const stickyTrigger = document.querySelector('.filament-sticky-trigger')
```

Comment:
We don't need JS for this, right?

Reply:
Yes we do. The intersection observer needs an element to watch. It's the only way to add a class that can be used to style elements when they are stuck. There's currently no way to target a stuck element in CSS.

Reply:
What exactly do we need that styling target for?

Sorry if I'm missing something, but the sticky header can be achieved with pure CSS, right? Or is there more to it?